components/sysutils/nmap/patches/01-climits.patch

```diff
@@ -0,0 +1,21 @@
+--- nmap-7.95/charpool.cc.orig	2024-02-28 20:46:45.000000000 +0200
```

Comment:
Hint: you can use `env - gmake refresh-patches` to normalize patches.

Reply:
thanks